zallet/src/config.rs

```diff
@@ -282,6 +288,115 @@ impl LimitsSection {
     }
 }
 
+/// Note management configuration section.
+///
+/// TODO: Decide whether this should be part of `[builder]`.
+#[derive(Clone, Debug, Default, Deserialize, Serialize, Documented, DocumentedFields)]
+#[serde(deny_unknown_fields)]
+pub struct NoteManagementSection {
+    /// The minimum value that Zallet should target for each shielded note in the wallet.
+    pub min_note_value: Option<u32>,
+
+    /// The target number of shielded notes with value at least `min_note_value` that
+    /// Zallet should aim to maintain within each account in the wallet.
+    ///
+    /// If an account contains fewer such notes, Zallet will split larger notes (in change
+    /// outputs of other transactions) to achieve the target.
+    pub target_output_count: Option<NonZeroU16>,
```

Comment:
It's probably a little confusing to have both `note` and `output` used more or less interchangeably in this section. We should either go with `min_note_value` and `target_note_count` or `min_output_value` and `target_output_count`.

Reply:
Resolution: for now we are not going to support transparent output management in Zallet, so as long as there are RPCs that enable wallet operators to manually mangage their outputs (e.g. `z_mergetoaddress` to combine transparent inputs), then that is sufficient for Zallet's target audience.